docs/design/audit-v2.md

```diff
@@ -0,0 +1,74 @@
+# Auditing V2: Random Node Selection
+
+## Abstract
+
+This design doc outlines how we will implement a Version 2 of the auditing service.
+With our current auditing service, we are currently auditing per segment of data.
+We need to keep this existing method of auditing, but also add an additional audit selection process that selects random nodes to audit.
+
+## Background
+
+As our network grows, it will take longer for nodes to get vetted.
+This is because every time an upload happens, we send 5% of the uploaded data to unvetted nodes and 95% to vetted nodes.
+When auditing occurs, we currently select a random stripe within a segment.
+If we're selecting audits at random per byte, every segment has some percentage that went to vetted nodes.
+As more nodes join the network, it will become exponentially less likely that an unvetted node will be audited since most data will be stored on vetted nodes.
+With a satellite with one petabyte of data currently, new nodes will take one month to get vetted, which is way too long.
+
+We want a way to select segments to audit based on statistically randomly selected storage nodes.
+However, the way we select nodes should also be biased toward unvetted nodes.
+Currently, there's not a way to select a stripe based on a storage node.
+
+## Design
+
+Two different loops will select audits:
+- One loop will select based on stripes.
+- The second loop will select based on nodes.
+
+After selection, the rest of the auditing process will occur the same way (picking a segment, picking a stripe, downloading all erasure shares associated with that stripe).
+The chances of selecting the same stripe are rare, but should be accounted for.
```

Comment:
We could use a bloom filter to determine whether we have audited a stripe before lol

Reply:
![tenor](https://user-images.githubusercontent.com/26885324/62400217-72392380-b54c-11e9-9521-1a4a6a6361d3.png)

Reply:
What's wrong if we occasionally audit a stripe more than once?

Reply:
I think it should be fine to audit a stripe more than once sometimes.